code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -534,6 +534,8 @@
 	return
 
 /obj/structure/trap/ctf/trap_effect(mob/living/L)
+	if(!is_ctf_target(L))
+		return
```

Comment:
Are we sure? Because it kills anyone WITHOUT the CTF faction, don't all mobs lack it?